src/main/java/nopol_examples/nopol_example_3/NopolExample.java

```diff
@@ -6,14 +6,13 @@
 	 * Return true if a is odd number
 	 */
 	public boolean isOddNumber(int a) {
-		int tmp = (a-1)%2;
+		int tmp = (a-1)%2; 
 		int foo = 0;
 		
-		if (tmp != 0) {
+		if (tmp < 0) { 
 			return true;
```

Comment:
This line (13) has been identified with a suspiciousness value of 26.73%.

<details>
     <summary>Failing tests that cover this line</summary>

- `nopol_examples.nopol_example_3.NopolExampleTest#test5`
- `nopol_examples.nopol_example_3.NopolExampleTest#test9`
</details>